www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: layouts/post.njk
+---
+
+After 40+ beta versions & release candidates we are so excited to introduce **Snowpack 2.0**, featuring:
+
+- A bundle-free dev environment that starts up in 50ms or less.
```

Comment:
I would aim to make a bullet-point list like this have shorter bullets for a cleaner glance-over. Details like "Webpack or Parcel", "for React, Preat, Vue, Svelte & more", etc, are more "between the fold" kind of details IMO